### PR TITLE
add readonly integration test with all modules

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: test/readonly/test.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  it('adds a new repository', query(`
+    INSERT INTO repository (repository_name, scan_on_push, image_tag_mutability)
+    VALUES ('test', false, 'MUTABLE');
+  `));
+
+  it('fails to apply', (done) => {
+    query(`
+      select * from iasql_apply();
+    `)((_e?: any) => done());  // Ignore failure
+  });
```
Comment (dfellis) [inline test/readonly/test.ts:62]: This doesn't actually confirm that it failed, though?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -56,6 +56,29 @@ jobs:
...
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run common tests
+        run: yarn coverage **/common/*
```
Comment (aguillenv) [inline .github/workflows/ci.yml:74]: this should be:
```suggestion
        run: yarn coverage **/readonly/*
```

Comment (aguillenv) [reply .github/workflows/ci.yml:74]: another option is that the read only test could live inside the `common` directory, we remove this new ci job and update the common one adding the read only credentials?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -56,6 +56,29 @@ jobs:
...
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run common tests
```
Comment (aguillenv) [inline .github/workflows/ci.yml:73]:
```suggestion
      - name: Run read-only tests
```

**Review by aguillenv — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

**Pull request merged**